packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -114,7 +114,7 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
         modelStreamID,
         streamId
       )
-    } else if (payload.data || payload.header.unique) {
```

Comment:
curious what payload.header.unique did before, how does this relate to set/unset?

Reply:
For the "single" account relation the `unique` field must not be set, but now with the "set" account relation it should be